src/components/add-locked-token/add-locked-token.scss

```diff
@@ -0,0 +1,15 @@
+.or {
```

Comment:
Optional nitfpick: We should probably name space this properly and perhaps give it a slightly better name, perhaps: `.add-locked-token-address__or-divider`